services/navigation/builtin/builtin.go

```diff
@@ -384,10 +408,17 @@ func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interfac
 				extra,
 			)
 			if err != nil {
+				svc.logger.Infof("hit motion error %s when navigating to waypoint %+v", err.Error(), wp)
```

Comment:
preference to wrap the error so that it reads "hit motion error when navigation to waypoint: {err}"

Reply:
done